### ISSUE TITLE
Incorrect parameter Alias definition in certain commands

### ISSUE THREAD

**Issue by AspenForester** [raandree/NTFSSecurity#36]
The following commands fail to import with Import-WinModule because of a bug with the definition of the parameter alias definition:
Add-NTFSAccess
Add-NTFSAudit
Get-NTFSAccess
Get-NTFSAudit
Get-NTFSOrphanedAccess
Get-NTFSOrphanedAudit
Get-NTFSSimpleAccess
Remove-NTFSAccess
Remove-NTFSAudit

The typical error raised is:
```PowerShell
Import-Module : Failed to generate proxies for remote module 'NTFSSecurity'. Proxy creation has been skipped for the 'Ad
d-NTFSAudit' command, because PowerShell could not verify the safety of a parameter alias name: 'IdentityReference, ID'.
At C:\Users\jole001\Documents\PowerShell\Modules\WindowsCompatibility\0.0.1\WindowsCompatibility.psm1:479 char:13
+             Import-Module  -Name $ImportNames -NoClobber:$NoClobber @ ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidData: (:) [Import-Module], InvalidOperationException
+ FullyQualifiedErrorId : ErrorSkippedUnsafeAliasName,Microsoft.PowerShell.Commands.ImportModuleCommand
```

This is further confirmed by @brucepay in [issue 38 of WindowsCompatibility](https://github.com/PowerShell/WindowsCompatibility/issues/38)

**Comment by raandree** [comment]
Thanks, and especially for the PR. I will merge it very noon!

**Comment by raandree** [comment]
Thanks @AspenForester, should all be fixed in 4.2.5 now.
